keras/layers/advanced_activations.py

```diff
@@ -208,3 +209,32 @@ def get_config(self):
         config = {'theta': float(self.theta)}
         base_config = super(ThresholdedReLU, self).get_config()
         return dict(list(base_config.items()) + list(config.items()))
+
+
+class SoftMax(Layer):
```

Comment:
Just `Softmax` would be more consistent with other names I think.

Reply:
Agreed: changed.